app/Policies/PostPolicy.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Rogue\Policies;
+
+use Rogue\Models\Post;
+use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class PostPolicy
+{
+    use HandlesAuthorization;
+
+    /**
+     * Create a new policy instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+    }
+
+    /**
+     * Determine if the given post can be seen by the user.
+     *
+     * @param  Illuminate\Contracts\Auth\Authenticatable $user
+     * @param  Rogue\Models\Post $post
+     * @return bool
+     */
+    public function show(Authenticatable $user, Post $post)
+    {
+        if ($post->status != 'accepted') {
```

Comment:
I think we usually do literal comparisons with `!==`

Reply:
ah thanks @sbsmith86 ! Updated :)